app/src/main/java/chat/rocket/android/infrastructure/SharedPreferencesRepository.kt

```diff
@@ -0,0 +1,21 @@
+package chat.rocket.android.infrastructure
+
+import android.content.Context
+import android.content.SharedPreferences
+
+class SharedPreferencesRepository(context: Context) : LocalRepository {
```

Comment:
on `AppModule` add a 
```
@Inject
fun provideSharedPreferenceRepository(preferences: SharedPreferences): LocalRepository {
    return  SharedPreferencesRepository(preferences)
}
```

and
```
class SharedPreferencesRepository(private val preferences: SharedPreferences) : LocalRepository {
```
  
So you can inject a `LocalRepository` into Interactors